Actions/VerifyPRChanges/VerifyPRChanges.ps1

```diff
@@ -6,25 +6,76 @@ Param(
   [Parameter(HelpMessage = "Head commit of the PR", Mandatory = $false)]
   [string] $headSHA,
   [Parameter(HelpMessage = "The name of the repository the PR is going to", Mandatory = $false)]
-  [string] $prBaseRepository
+  [string] $prBaseRepository,
+  [Parameter(HelpMessage = "The id of the pull request", Mandatory = $false)]
+  [string] $pullRequestId
 )
 
+function ValidateFiles
+(
+  [Object[]] $Files
+)
+{
+  $Files | ForEach-Object {
+    $filename = $_.filename
+    $status = $_.status
+    Write-Host "- $filename $status"
+    $extension = [System.IO.Path]::GetExtension($filename)
+    $name = [System.IO.Path]::GetFileName($filename)
+    if ($extension -eq '.ps1' -or $extension -eq '.yaml' -or $extension -eq '.yml' -or $name -eq "CODEOWNERS" -or $filename.StartsWith(".github/")) {
+      throw "Pull Request containing changes to scripts, workflows or CODEOWNERS are not allowed from forks."
+    }
+  } 
+}
+
+function ValidatePullRequest
+(
+  [string[]] $PullRequestRepository,
+  [string[]] $PullRequestId,
+  [object] $Headers
+)
+{
+  $url = "https://api.github.com/repos/$($prBaseRepository)/pulls/$pullRequestId"
+  $pullRequestDetails = Invoke-WebRequest -UseBasicParsing -Headers $Headers -Uri $url | ConvertFrom-Json
+
+  if ($pullRequestDetails.changed_files -gt 1000) {
```

Comment:
Wouldn't you need to check added files + changed files?
![image](https://github.com/microsoft/AL-Go/assets/10775043/fff5d542-892e-49e9-943c-e03d2752f322)

Reply:
I've tested this on a couple of PRs now. It seems like additions and deletions is referring to changes within the files. See for example this PR
https://github.com/microsoft/ALAppExtensions/pull/16096
<img width="220" alt="image" src="https://github.com/microsoft/AL-Go/assets/117829001/1e775d90-ec99-4b21-998d-2aad20c119b8">

Reply:
Ahh - so changed_files contains all changed, deleted or added files...